app/services/data/get-app-workloads.js

```diff
@@ -0,0 +1,43 @@
+const knexConfig = require('../../../knexfile').development
+const knex = require('knex')(knexConfig)
+const CaseDetails = require('wmt-probation-rules').CaseDetails
+const OmWorkload = require('wmt-probation-rules').OmWorkload
+
+module.exports = function (initialId, batchSize) {
+  var maxId = initialId + batchSize
```

Comment:
Think this might be best done outside of here so we're just passing in one parameter of range [nextId, lastId] kind of style. What do you guys think?